Source/KSCrash/Source/KSCrash/Recording/Sentry/BSG_KSCrashSentry_Deadlock.m

```diff
@@ -114,7 +114,7 @@ - (void)handleDeadlock {
     bsg_g_context->registersAreValid = false;
 
     BSG_KSLOG_DEBUG(@"Calling main crash handler.");
-    bsg_g_context->onCrash();
+    bsg_g_context->onCrash('e', "");
```

Comment:
Do we want to return something that indicates this is a deadlock rather than an empty string?

Reply:
This sentry is unused.